includes/displayBikes.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function printCards (array $allBikes){
```

Comment:
Great function, but this should be unit tested!

Reply:
Rather than echoing the bike directly, in order to make this testable you'll need to change this so it's returning the whole output string (all the html for all the bikes) instead.